libs/chlorophyll/scss/components/accordion/_mixins.scss

```diff
@@ -89,4 +90,20 @@
       }
     }
   }
+
 }
+
+@mixin accordion-list-item-content() {
+  overflow: hidden;
+  transition: height 200ms linear, opacity 200ms linear;
+  margin-left: 1em;
+
+  &.content-hidden {
```

Comment:
this will be controlled by the hidden attribute.

Reply:
Removed.